platform/openide.windows/src/org/openide/windows/WindowManager.java

```diff
@@ -741,6 +741,26 @@ public String getRole() {
         return null;
     }
     
+    /**
+     * Creates a Mode from the ModeConfig XML.
+     * @param xml ModeConfig XML
```

Comment:
Use `@since`. Modify version in `manifest.mf`, update `apichanges.xml` accordingly. Here is some [outdated info](http://wiki.netbeans.org/APIDevelopment).